packages/flutter_tools/test/general.shard/version_test.dart

```diff
@@ -376,19 +382,19 @@ void main() {
         final FlutterVersion flutterVersion = FakeFlutterVersion(
           repositoryUrl: flutterNonStandardUrlDotGit,
         );
-        expect(() => VersionUpstreamValidator(version: flutterVersion).run(), returnsNormally);
+        final FakePlatform platform = createTestPlatform(flutterGitUrl: flutterNonStandardUrlDotGit);
+        expect(() => VersionUpstreamValidator(version: flutterVersion, platform: platform).run(), returnsNormally);
       }, overrides: <Type, Generator> {
-        Platform: () => FakePlatform(environment: <String, String> {
-          'FLUTTER_GIT_URL': flutterNonStandardUrlDotGit,
-        }),
+        Platform: () => createTestPlatform(flutterGitUrl: flutterNonStandardUrlDotGit),
```

Comment:
this context platform never gets called in the test, right?

Reply:
Unless the test doesn't call `globals.flutterGit`, which it definitely does :)